explorer/site-config/anvil/images/favicons/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "",
+  "short_name": "",
+  "icons": [
+    {
```

Comment:
@BruceRodrigues same comment here. Can we avoid having multiple copies of this file?